lib/app/shared/presentation/widgets/layout/home_nav_bar.dart

```diff
@@ -49,32 +50,43 @@ class _HomeNavBarState extends State<HomeNavBar> {
                           style: TextButton.styleFrom(padding: EdgeInsets.zero),
                           child: AnimatedSwitcher(
                             duration: const Duration(milliseconds: 500),
-                            child: Theme.of(context).brightness == Brightness.light
-                                ? Image.asset(
-                                    AppImages.logoDark,
-                                    width: 130,
-                                  )
-                                : Image.asset(
-                                    AppImages.logoLight,
-                                    width: 130,
-                                  ),
+                            child:
+                                Theme.of(context).brightness == Brightness.light
+                                    ? Image.asset(
+                                        AppImages.logoDark,
+                                        width: 130,
+                                      )
+                                    : Image.asset(
+                                        AppImages.logoLight,
```

Comment:
`final isDarkTheme = Theme.of(context).brightness == Brightness.dark;`

Was used on a class bellow and could help to do:

`Image.asset( isDarkTheme ? AppImages.logoDark : AppImages.logoLight,
width: 130,
)`